src/app/apis/covid.js

```diff
@@ -0,0 +1,40 @@
+const LOAD = 'reactcapstone/apis/LOAD';
+
+const init = [
+];
+
+export function load(country) {
+  return {
+    type: LOAD,
+    country,
+  };
+}
+
+const getApiData = async () => {
+  const res = await fetch('https://disease.sh/v3/covid-19/countries/');
+  const data = await res.json();
+  return data;
+};
+
+export const displayCountries = () => async (dispatch) => {
+  const countries = await getApiData();
+
+  const countriesArray = countries.map((ele) => ({
+    // eslint-disable-next-line no-underscore-dangle
```

Comment:
- [ ] please avoid turning off eslint, linters are there to make your code more efficient and readable.